l5kit/l5kit/sampling/agent_sampling.py

```diff
@@ -113,9 +111,9 @@ def generate_agent_sample(
             )[0]
         except IndexError:
             raise ValueError(f" track_id {selected_track_id} not in frame or below threshold")
-        agent_centroid = agent["centroid"]
-        agent_yaw = float(agent["yaw"])
-        agent_extent = agent["extent"]
+        agent_centroid_m = agent["centroid"]
+        agent_yaw_rad = float(agent["yaw"])
+        agent_extent_m = agent["extent"]
         selected_agent = agent
```

Comment:
what does _m stand for? might be unclear naming

Reply:
it stands for `meter`

Reply:
metres :)